src/anubhav/scannerexamples/CalculatorScanner.java

```diff
@@ -0,0 +1,64 @@
+package anubhav.scannerexamples;
+
+import java.util.Scanner;
+
+public class CalculatorScanner {
+
+	double numA;
+	double numB;
+	double answer;
+	String operation; 
+
+	void calculator(String str){
+		str= operation;
+		switch (str) {
```

Comment:
you should not handle this bunch of conditions,
you should give some menu driven like

1 --> addition
2 --> sub
3 --> multi
4 --> div

chose number and pass this number to switch case, using this way you not require to write bunch of lines.